web/client/utils/PluginsUtils.js

```diff
@@ -273,20 +337,24 @@ const PluginsUtils = {
         return {
             id: id || name,
             name,
-            impl: includeLoaded(name, loadedPlugins, impl.loadPlugin || impl.displayName || impl.prototype.isReactComponent ? impl : impl(stateSelector)),
+            impl: includeLoaded(name, loadedPlugins, getPluginImplementation(impl, stateSelector)),
             cfg: assign({}, impl.cfg || {}, isObject(pluginDef) ? parsePluginConfig(state, plugins.requires, pluginDef.cfg) : {}),
             items: getPluginItems(state, plugins, pluginsConfig, name, id, isDefault, loadedPlugins)
         };
     },
     getPluginItems,
-    getConfiguredPlugin: (pluginDef, loadedPlugins, loaderComponent) => {
+    getConfiguredPlugin: (pluginDef, loadedPlugins = {}, loaderComponent) => {
         if (pluginDef) {
```

Comment:
if plugin is initialized with a string in localConfig.json the cfg in pluginDef contains chars of the name
```
{
    "name": "Search",
    "position": 1,
    "priority": 1,
    "cfg": {
        "0": "S",
        "1": "e",
        "2": "a",
        "3": "r",
        "4": "c",
        "5": "h"
    },
    "items": []
}
```

Reply:
@allyoucanmap the previous fix should also fix this one